server/files/serializers.py

```diff
@@ -1,9 +1,85 @@
 from rest_framework import serializers
 
-from .models import File
+from ..notebooks.models import Notebook
+from .models import File, FileSource, FileUpdateOperation
 
 
 class FilesSerializer(serializers.ModelSerializer):
+    """
+    Gives a summary of file (not its content)
+    """
+
     class Meta:
         model = File
         fields = ("id", "notebook_id", "filename", "last_updated")
+
+
+class FileSourceSerializer(serializers.ModelSerializer):
```

Comment:
Should have a version of this serializer for the not-logged-in case, without the url field